### ISSUE TITLE
malware..

### ISSUE THREAD

**Issue by israel-porras** [Dashbloxx/MultiRoblox#5]
![image](https://github.com/user-attachments/assets/00970197-b40a-43bc-9325-a61c9110c56c)

virustotal says it all..

![image](https://github.com/user-attachments/assets/5e0cfdfc-5147-4d5f-8cbb-6fc3b2e2f952)
![image](https://github.com/user-attachments/assets/e8a16008-72be-4bd7-9721-9c6a669b777e)

![image](https://github.com/user-attachments/assets/ce516588-326c-4c24-93d5-77d2ddc2e0b7)
so tell ,me.. is this malware. or am i just fucking tripping?

**Comment by ammystic** [comment]
not malware.. read through the source.

**Comment by GangstaPichu** [comment]
Virus Total isn't always accurate. Especially if you don't pay for a license or send it into them to mark it off their list. There are false positives with these websites.

**Comment by Realswitzer** [comment]
hello yes this one engine that nobody has heard of detected virus

**Comment by florrika** [comment]
use triage, no one cares if that unknown antivirus flagged it

**Comment by LeoDev5** [comment]
Unknown Antivirus and it's false positive flagged. nevermind.
Why are you saying its a mawlare with 9/94 flagged ?
You're only argument is a real microsoft link launched by VirusTotal when their scanning the file.

**Comment by ghost** [comment]
Yeah, Virustotal doesn't always get scans right. Or mostly not. But still this IS A FUCKING VIRUS!! WHAT RETARD DOESN'T KNOW THAT THEIR LOCATION IS GETTING LOGGED WHILE PLAYING ROBLOX ON TWO INSTANCES! SHIT BEFORE I EVEN KNEW ABOUT THIS CMD GUI TROJAN, RANSOMWARE, MALWARE SHIT I DOWNLOADED ONE AND GOT A RANSOMWARE-

**Comment by Dev7z** [comment]
https://github.com/Dashbloxx/MultiRoblox/issues/20#issuecomment-3690386544 im just farming issues at this point
